src/content/recipes/less.md

```diff
@@ -16,39 +16,21 @@ How to setup Less and Storybook
 
 Less is a popular CSS preprocessor that allows developers to write more maintainable and reusable stylesheets. Storybook is an industry-standard tool for developing and testing UI components in isolation. With the help of the `@storybook/addon-styling` package, developers can easily incorporate Less stylesheets into their Storybook components.
 
-## Adding `@storybook/addon-styling`
+## Before you begin
 
-<div class="aside aside__no-top">
+If you are using Vite as your storybook builder, all you need to do is import your Less files into your `preview.js` file.
```

Comment:
storybook → Storybook (saw this a couple other places, as well)

Reply:
Good catch, Gach! Fixed 😁